src/Downloader.php

```diff
@@ -13,7 +13,10 @@ public static function downloadCertificateFromUrl(string $url, int $timeout = 30
 
         $streamContext = stream_context_create([
             'ssl' => [
+                'verify_peer'       => false,
```

Comment:
As I've been looking into the testing of SSL chains I discovered this will need to be the default for certain SSLs, as with it set to true we cannot connect to check the SSL info to begin with.

Additionally, with this line excluded we are not actually able to do any SSL expiration checking with the library; you cannot connect to the server if the SSL is expired. So in a sense the isValid method will **always** return as true when not providing a domain to check.

That said, if this line is not added then you may as well remove this check from SSLcertificate.php:
```
        if (!Carbon::now()->between($this->validFromDate(), $this->expirationDate())) {
            return false;
        }
```
As it will never be triggered for an expired SSL since you will get hit with a CouldNotDownloadCertificate exception before we'd ever reach this.